# 电机反馈消息
float64 pos_fdb  # 电机位置反馈
float64 spd_fdb  # 电机速度反馈
float64 tor_fdb  # 电机力矩反馈